ivre/xmlnmap.py

```diff
@@ -1282,7 +1282,7 @@ def __init__(self, fname, categories=None, source=None,
         self._collection = self._db.nmap.db[self._db.nmap.colname_hosts]
         self._scancollection = self._db.nmap.db[self._db.nmap.colname_scans]
         if gettoarchive is None:
-            self._gettoarchive = lambda c, a, s: []
+            self._gettoarchive = lambda a, s: []
```

Comment:
That's only so that `NmapHandler` and `Nmap2Mongo` constructors have the same prototype.